src/utils/test-runner.js

```diff
@@ -64,6 +64,10 @@ async function testRunner(argv) {
 
         shell.runCommand(command, commandArgs);
     } else {
+        // Jest will not set the env if not run from the bin executable
+        if (process.env.NODE_ENV == null) {
```

Comment:
If an environment variable is not set its value is always `undefined` and not `null`.

```suggestion
        if (process.env.NODE_ENV === undefined) {
```

Reply:
This is the exact code Jest uses. I want to keep parity between the two modes. It's very confusing to get a failure in a test and then try to debug and get different results.